.gitignore

```diff
@@ -1,5 +1,5 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
+.env.local
```

Comment:
i believe .gitignore already has `.env.local`, so could we remove this to keep things organized?